### PR TITLE
54 custom donation block

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This pull request makes the following changes:

- Adds a donation block
- Removes a 1px-wide transparent border on the right-hand side of `img.footer-skyline.png` that was causing a very thin grey line seen in some of these screenshots

Here's the controls: 

Screen Shot 2019-11-11 at 13 15 18

Set the headline, CTA text, button text, and button link. I'm using sample text sourced from https://www.damninteresting.com/this-place-is-not-a-place-of-honor/

Here's how it displays at various sizes. Note that the left-aligned version will bump out to the left, to match the mockups. No other blocks have this behavior.

Screen Shot 2019-11-11 at 12 40 19
Screen Shot 2019-11-11 at 12 40 24
Screen Shot 2019-11-11 at 12 40 32
Screen Shot 2019-11-11 at 12 40 44
Screen Shot 2019-11-11 at 12 40 50

This last image shows hover styles for the button, which are the site's default styles for `.btn.btn-primary`.

## Why

For #54

## Testing/Questions

Features that this PR affects:

- places that use blocks

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] The center-aligned block looks kinda odd; should we center its text?
- [ ] I've only tested this in the two-column template; do we want to test it on the single-column template?

Steps to test this PR:

1. In a post, add a "City Limits Donation Block" block to the post.
2. Configure its headline, copy, button and link.
3. Update the post
4. View the post

### CODE REVIEW

**Comment by joshdarby** [comment]
> The center-aligned block looks kinda odd; should we center its text?

It does look kind of weird. Instead of centering the text, could we just remove the width constraints and let the text fill the entire container like this?

Screen Shot 2019-11-11 at 4 04 55 PM

**Comment by joshdarby** [comment]
@benlk The center aligned version stretches the city skyline image across the entire container, but on the default version (no alignment selected) the city skyline isn't stretched across the entire container even though it's the same size as the center aligned version.

Centered version:
Screen Shot 2019-11-11 at 4 06 15 PM

No alignment version:
Screen Shot 2019-11-11 at 4 06 02 PM

**Comment by joshdarby** [comment]
Also, this isn't really an issue, but something interesting to note:

Apparently the Gutenberg gives the same class on the editor as it does on the frontend, which in this case is `wp-block-citylimits-citylimits-custom-donate`, causing the city skyline to appear in the editor for the block 😄 

(background darkened so you can see it easier)
Screen Shot 2019-11-11 at 4 11 47 PM

**Comment by benlk** [comment]
Those should all be resolved in 7a883e2.

File: wp-content/themes/citylimits/blocks/citylimits-custom-donate/style.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  font-size: 20px;
+}
+.wp-block-citylimits-citylimits-custom-donate .border {
+  margin: 0.75rem;
+  border: 1px solid white;
+  padding: 1.5rem 1.5rem 50px;
+  clear: both;
+}
+.wp-block-citylimits-citylimits-custom-donate .border {
+  max-width: 20em;
```
Comment (joshdarby) [inline wp-content/themes/citylimits/blocks/citylimits-custom-donate/style.css]: @benlk The `max-width` is causing issues now on the default variant with no alignment :/

Screen Shot 2019-11-11 at 4 39 57 PM

Comment (benlk) [reply wp-content/themes/citylimits/blocks/citylimits-custom-donate/style.css]: Give https://github.com/INN/umbrella-citylimits/pull/104/commits/c3fe29224c6802136eec08908bafff254084a0a1 a try?

Comment (joshdarby) [reply wp-content/themes/citylimits/blocks/citylimits-custom-donate/style.css]: Looks great!

**Pull request merged**